website/docs/r/storage_notification.html.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: "google"
+page_title: "Google: google_storage_notification"
+sidebar_current: "docs-google-storage-notification"
+description: |-
+  Creates a new notification configuration on a specified bucket.
+---
+
+# google\_storage\_notification
+
+Creates a new notification configuration on a specified bucket, establishing a flow of event notifications from GCS to a Cloud Pub/Sub topic.
+ For more information see 
+[the official documentation](https://cloud.google.com/storage/docs/pubsub-notifications) 
+and 
+[API](https://cloud.google.com/storage/docs/json_api/v1/notifications).
+
+## Example Usage
+
+```hcl
+resource "google_storage_bucket" "bucket" {
+	name = "default_bucket"
+}
+		
+resource "google_pubsub_topic" "topic" {
+	name = "default_topic"
+}
+
+//GCS default storage account must have "projects.topics.publish" permission
```

Comment:
can we refer to Google's docs here? And a specific example of this special account will greatly help here. Is it <projectId>@gs-project-accounts.iam.gserviceaccount.com?